### PR TITLE
hack-athens23: Add guest talks

### PR DESCRIPTION
**Pull request by razvand**
Add guest talks.

### CODE REVIEW

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 1]
```diff
@@ -43,21 +43,23 @@ As part of the Unikraft community [Răzvan Deaconescu](https://github.com/razvan
...
+| 10:15 - 11:15 | Introduction to Unikraft: Building and Running                       |                    |
 | 11:15 - 11:30 | Break                                                                |                    |
 | 11:30 - 12:45 | Unikraft Internals: Building, Configuring, Using Libraries           |                    |
 | 12:45 - 14:00 | Lunch                                                                |                    |
 | 14:00 - 15:00 | Running (Complex) Applications in Unikraft                           |                    |
 | 15:00 - 16:00 | Porting Applications to Unikraft                                     |                    |
 | 16:00 - 16:15 | Break                                                                |                    |
-| 16:15 - 17:45 | Guest Talks                                                          |                    |
+| 16:15 - 16:45 | The Value of Unikernels <br /> in the Emerging Disaggregated HPC and AI Clusters | Dimitris Syrivelis |
+| 16:45 - 17:15 | Unikernels for Serverless                                            | Anastassios Nanos  |
```
[the remark this answers is not in the document]
Comment (razvand) [reply content/en/community/hackathons/2023-03-athens/_index.md:57]: No, that's it.

**Review by StefanJum — APPROVED** [review]
All good.
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Comment by razvand** [comment]
> Looks good. A few things that can be improved:
> 
>     * Prepend title with "Guest Talk: "
> 
>     * Add affiliation next to the speakers' names (Nubificus, NVIDIA, ...)
> 
>     * Fill in the Presenter field in the rest of the sessions.
> 
> 
> If you think these are unnecessary please feel free to merge as is; I have approved already slightly_smiling_face

@michpappas, done. Re-approve if OK, and then I'll publish the information.

**Review by michpappas — CHANGES_REQUESTED** [review]
IIRC Dimitris Syrivelis is from NVIDIA, not UoA: 
https://www.linkedin.com/in/dimitris-syrivelis-3208533
We can double-check with Anastassios if needed.

Other than that looks great 👍

**Comment by razvand** [comment]
> IIRC Dimitris Syrivelis is from NVIDIA, not UoA: https://www.linkedin.com/in/dimitris-syrivelis-3208533 We can double-check with Anastassios if needed.
> 
> Other than that looks great +1

@michpappas, done. See if this is OK now.

**Review by michpappas — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**